src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -292,39 +290,13 @@ func (s *cloudService) CreateVM(ctx context.Context, req *pb.CreateVMRequest) (r
 		}
 	}
 
-	if s.aaKBCParams != "" { // Keep AA_KBC_PARAMS support as it is used by e2e test, KBS is dynamic k8s service in e2e test
```

Comment:
I'm wondering if it's easier to adapt this code to add AA config toml via configmap? 
Please note that my comments is to kickstart the discussion as I believe some these code will be reused. It's perfectly fine to handle this in a separate PR. But since you are removing these code, I thought of at least checking if it makes sense to adapt the code instead of deleting and re-adding..

Reply:
@bpradipt It's currently renaming from ` s.aaKBCParams` to ` s.initdata`.